src/index.ts

```diff
@@ -251,4 +259,14 @@ export {
   WordWrapIcon,
   ZoomInIcon,
   ZoomOutIcon,
+  DropdownButton,
```

Comment:
chöntisch do echt no d'Exports sortiere? 
https://stackoverflow.com/questions/69115477/how-can-i-order-alphabetically-a-list-in-plain-text-in-vs-code#:~:text=Press%20Ctrl%20%2B%20A%20to%20select,%22Sort%20lines%20Descending%22%20accordingly.